assets/js/WfParameter.js

```diff
@@ -277,11 +277,48 @@ export default class WfParameter extends React.Component {
     }
   }
 
+  displayConditionalUI(condition, type) {
+    // Checks if conditional UI should be displayed
+    // type is either 'visible_if' or 'visible_if_not'
+    if(('id_name' in condition) && ('value' in condition)) {
+      var condValues = condition['value'].split('|').map(cond => cond.trim());
+      var selectionIdx = parseInt(this.props.getParamText(condition['id_name']));
+      if(selectionIdx != NaN) {
+        var menuItemsStr = this.props.getParamMenuItems(condition['id_name']);
+        if(menuItemsStr) {
+          var menuItems = menuItemsStr.split('|').map(cond => cond.trim());
```

Comment:
Is there any case where you would want the raw string instead of the array from getParamMenuItems? If not, this split could be moved into getParamMenuItems.